src/options.js

```diff
@@ -200,6 +203,47 @@ const OptionHandler = (function () {
 
         case "debugMode":
             Logger.setDebugMode(optionValue);
+            break;
+
+        case "qrColor":
+        case "qrBackgroundColor": {
+            const elQrColor = document.getElementById("qrColor");
+            const elQrBackgroundColor = document.getElementById("qrBackgroundColor");
+
+            const qrCodeColor = Colors.hexToRgb(elQrColor.value);
+            const qrCodeBackgroundColor = Colors.hexToRgb(elQrBackgroundColor.value);
+
+            const colorContrast = Colors.contrastRatio(qrCodeColor, qrCodeBackgroundColor);
+
+            const actionButton = {
+                text: "messageAutoSelectColorButton",
+                action: () => {
+                    const invertedColor = Colors.invertColor(qrCodeColor);
+                    browser.storage.sync.set({
+                        "qrBackgroundColor": invertedColor
+                    }).catch((error) => {
+                        Logger.logError("could not save option", option, ": ", error);
+                        MessageHandler.showError("couldNotSaveOption", true);
+                    }).finally(() => {
+                        applyOptionLive();
+                        elQrBackgroundColor.value = invertedColor;
+                    });
+                }
+            };
+
+            // breakpoints: https://github.com/rugk/offline-qr-code/pull/86#issuecomment-390426286
```

Comment:
Oh and once we are at it. You can link to my comment, yes, but please still add comments for the colors, i.e. the accessibility guidelines I mentioned.

Or even better remember the "no magic numbers" rules. So put these numbers into constants. As they are color stuff not (only) related to the options handler, the best would be to use these as constants in the `Colors` module, i.e. make them "public constants". (add to `me` in that module)